doc/api/crypto.md

```diff
@@ -622,7 +622,7 @@ provided, `otherPublicKey` is expected to be a [`Buffer`][],
 If `outputEncoding` is given a string is returned; otherwise, a
 [`Buffer`][] is returned.
 
-### diffieHellman.generateKeys([encoding])
+### diffieHellman.generateKeys(\[encoding\])
```

Comment:
These were rendering as links in the heading because of the reference link `[encoding]`